### ISSUE TITLE
Bounce animation changing div rotation

### ISSUE THREAD

**Issue by Korysam15** [daneden/animate.css#804]
All,

I am currently trying to apply the bounce animation to a couple of rotated divs on hover, however when I hover over the rotated divs, they change orientation back to 0 degrees. Below is my css code:

```
.rotate-90 {
	/* Safari */
	-webkit-transform: rotate(-90deg);
	/* Firefox */
	-moz-transform: rotate(-90deg);
	/* IE */
	-ms-transform: rotate(-90deg);
	/* Opera */
	-o-transform: rotate(-90deg);
	/* Internet Explorer */
	filter: progid:DXImageTransform.Microsoft.BasicImage(rotation=3);
}
```

Here is the div css:

```
#left {
	position: absolute;
	left: 0;
	top: 50vh;
	text-align: center;
}
```

Here is the html code:

```
<div id="left" class="rotate-90">
  <a href="portfolio.html">
  <i class="fa fa-angle-double-up"></i>
  <br>
  PORTFOLIO
  </a>
</div>
```

Any suggestions would be appreciated! Thank you

**Comment by WarenGonzaga** [comment]
Can you provide an issue using codepen.io so we can quickly figure it out how to fix it?

**Comment by Korysam15** [comment]
https://codepen.io/KKUDDA/pen/MVqpvb

**Comment by WarenGonzaga** [comment]
Are you trying to use bounce and rotate animation at the same time?

**Comment by Korysam15** [comment]
Yes. I was wanting it to bounce while keeping the same orientation (i.e. -90 degree or 90 degree rotation).

**Comment by eltonmesquita** [comment]
You can't set two animations at the same time to the same element, as like any property in CSS all you get is to override the previous declaration. If the element has a transform declared it'll also override it as animate uses transform for animations.
The way to solve it is to put the element inside a parent element and set one animation to the parent and the other one to the element itself. The same is true to the transformed element + animation.

**Comment by WarenGonzaga** [comment]
@Korysam15 now let's close the issue :)
